pages/docs/reference/reflection.md

```diff
@@ -11,13 +11,57 @@ Reflection is a set of language and library features that allows for introspecti
 Kotlin makes functions and properties first-class citizens in the language, and introspecting them (i.e. learning a name or 
 a type of a property or function at runtime) is closely intertwined with simply using a functional or reactive style.
 
-> On the Java platform, the runtime component required for using the reflection features is distributed as a separate
-JAR file (`kotlin-reflect.jar`). This is done to reduce the required size of the runtime library for applications
-that do not use reflection features. If you do use reflection, please make sure that the .jar file is added to the
-classpath of your project.
+> On the JavaScript platform, only class references are currently supported. [Learn more about reflection in Kotlin/JS](js-reflection.html).
 {:.note}
 
-## Class References
+## JVM dependency
+
+On the JVM platform, the runtime component required for using the reflection features is distributed as a separate
+artifact. This is done to reduce the required size of the runtime library for applications
+that do not use reflection features. If you do use reflection, add the dependency on `kotlin-reflect` to your project:
+* In Gradle:
+  <div class="multi-language-sample" data-lang="groovy">
+  <div class="sample" markdown="1" theme="idea" mode="groovy" data-highlight-only>
+  
+  ```groovy
+  dependencies {
+      compile "org.jetbrains.kotlin:kotlin-reflect:{{ site.data.releases.latest.version }}"
```

Comment:
AFAIK `compile` is deprecated, do we want to promote it? Maybe `implementation`, as in the kts version?